### DISCUSSION TITLE
How do I remove the @ mention from bot replies?

### DISCUSSION THREAD

**Discussion by chronz** [eritislami/evobot#834] [Q&A]
Just wanted to remove the mentions from bot replies as it might get annoying sometimes. Which file is it located in?

**Comment by SrIzan10** [comment]
delete <@{author}> from your locale file

**Reply by chronz** [reply]
Thanks for the reply! I have removed all of those in the locale file, but still end up with some messages with mentions.
Specifically:
@example, You need to join a voice channel first!
@example, There was an error executing that command.

I'm thinking that those may be from a separate file?

**Reply by SrIzan10** [reply]
what ide do you use? for example vscode...

**Reply by chronz** [reply]
im using vscode

**Reply by SrIzan10** [reply]
ok, search in vscode You need to join a voice channel first! and There was an error executing that command. and delete the author array from all of them.
